<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyNotePad</class>
 <widget class="QMainWindow" name="MyNotePad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1119</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyNotePad</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1119</width>
     <height>25</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuOpen_Recent_Files">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>Open Recent Files</string>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="title">
      <string>Open Recent Files</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>Icons/Recent.png</normaloff>Icons/Recent.png</iconset>
     </property>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="menuOpen_Recent_Files"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionFind"/>
    <addaction name="actionFind_Next"/>
    <addaction name="separator"/>
    <addaction name="actionZoom_In"/>
    <addaction name="actionZoom_Out"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_All"/>
    <addaction name="actionClear_All"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuFormat">
    <property name="title">
     <string>Format</string>
    </property>
    <widget class="QMenu" name="menuBullets">
     <property name="toolTip">
      <string>Create Bulleted List</string>
     </property>
     <property name="title">
      <string>Bullets</string>
     </property>
     <addaction name="actionDisc"/>
     <addaction name="actionCircle"/>
     <addaction name="actionSquare"/>
    </widget>
    <widget class="QMenu" name="menuNumbering">
     <property name="toolTip">
      <string>Create Numbered List</string>
     </property>
     <property name="title">
      <string>Numbering</string>
     </property>
     <addaction name="actionArabic"/>
     <addaction name="actionUppercase_Roman"/>
     <addaction name="actionLowercase_Roman"/>
     <addaction name="actionUppercase_Alphabetic"/>
     <addaction name="actionLowercase_Alphabetic"/>
    </widget>
    <addaction name="actionBold"/>
    <addaction name="actionItalic"/>
    <addaction name="actionUnderline"/>
    <addaction name="actionFont"/>
    <addaction name="separator"/>
    <addaction name="actionStrikethrough"/>
    <addaction name="actionSubscript"/>
    <addaction name="actionSuperscript"/>
    <addaction name="separator"/>
    <addaction name="menuBullets"/>
    <addaction name="menuNumbering"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionColour"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionToolbar"/>
    <addaction name="actionStatus_Bar"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuFormat"/>
   <addaction name="menuView"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_As"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionBold"/>
   <addaction name="actionItalic"/>
   <addaction name="actionUnderline"/>
   <addaction name="actionFont"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionFind"/>
   <addaction name="actionFind_Next"/>
   <addaction name="separator"/>
   <addaction name="actionZoom_In"/>
   <addaction name="actionZoom_Out"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/ico/Icons/New.png</normaloff>:/ico/Icons/New.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="statusTip">
    <string>Create New Document</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/ico/Icons/Open.png</normaloff>:/ico/Icons/Open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="statusTip">
    <string>Open Documents</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/ico/Icons/Save.png</normaloff>:/ico/Icons/Save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="statusTip">
    <string>Save Document</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/ico/Icons/Save_As.png</normaloff>:/ico/Icons/Save_As.png</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="statusTip">
    <string>Save Document Using New Name</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/ico/Icons/Cancel.png</normaloff>:/ico/Icons/Cancel.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="statusTip">
    <string>Exit MyNotePad</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/ico/Icons/Cut.png</normaloff>:/ico/Icons/Cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="statusTip">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/ico/Icons/Copy.png</normaloff>:/ico/Icons/Copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="statusTip">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/ico/Icons/Paste.png</normaloff>:/ico/Icons/Paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="statusTip">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/ico/Icons/Undo.png</normaloff>:/ico/Icons/Undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="statusTip">
    <string>Undo What You Did</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/ico/Icons/Redo.png</normaloff>:/ico/Icons/Redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="statusTip">
    <string>Redo What You Did</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/ico/Icons/Find.png</normaloff>:/ico/Icons/Find.png</iconset>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
   <property name="statusTip">
    <string>Find What You Want</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionFind_Next">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/ico/Icons/Find Next.png</normaloff>:/ico/Icons/Find Next.png</iconset>
   </property>
   <property name="text">
    <string>Find Next</string>
   </property>
   <property name="statusTip">
    <string>Find Next</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/ico/Icons/Zoom In.png</normaloff>:/ico/Icons/Zoom In.png</iconset>
   </property>
   <property name="text">
    <string>Increase Font Size</string>
   </property>
   <property name="statusTip">
    <string>Zoom In</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/ico/Icons/Zoom Out.png</normaloff>:/ico/Icons/Zoom Out.png</iconset>
   </property>
   <property name="text">
    <string>Decrease Font Size</string>
   </property>
   <property name="statusTip">
    <string>Zoom Out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/ico/Icons/Select All.png</normaloff>:/ico/Icons/Select All.png</iconset>
   </property>
   <property name="text">
    <string>Select All</string>
   </property>
   <property name="statusTip">
    <string>Select All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionClear_All">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/ico/Icons/Clear.png</normaloff>:/ico/Icons/Clear.png</iconset>
   </property>
   <property name="text">
    <string>Clear All</string>
   </property>
   <property name="statusTip">
    <string>Clear All</string>
   </property>
  </action>
  <action name="actionBold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/ico/Icons/Bold.png</normaloff>:/ico/Icons/Bold.png</iconset>
   </property>
   <property name="text">
    <string>Bold</string>
   </property>
   <property name="statusTip">
    <string>Bold Text</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionItalic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/ico/Icons/Italic.png</normaloff>:/ico/Icons/Italic.png</iconset>
   </property>
   <property name="text">
    <string>Italic</string>
   </property>
   <property name="statusTip">
    <string>Italic Text</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionUnderline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/ico/Icons/Underline.png</normaloff>:/ico/Icons/Underline.png</iconset>
   </property>
   <property name="text">
    <string>Underline</string>
   </property>
   <property name="statusTip">
    <string>Underline Text</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/ico/Icons/Font.png</normaloff>:/ico/Icons/Font.png</iconset>
   </property>
   <property name="text">
    <string>Font</string>
   </property>
   <property name="statusTip">
    <string>Choose Font</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionColour">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/ico/Icons/Colour Palette.png</normaloff>:/ico/Icons/Colour Palette.png</iconset>
   </property>
   <property name="text">
    <string>Colour</string>
   </property>
   <property name="statusTip">
    <string>Choose Colour</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/ico/Icons/About.png</normaloff>:/ico/Icons/About.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="statusTip">
    <string>About MyNotePad</string>
   </property>
  </action>
  <action name="actionSubscript">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/ico/Icons/Subscript.png</normaloff>:/ico/Icons/Subscript.png</iconset>
   </property>
   <property name="text">
    <string>Subscript</string>
   </property>
   <property name="statusTip">
    <string>Subscript</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="actionSuperscript">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/ico/Icons/Superscript.png</normaloff>:/ico/Icons/Superscript.png</iconset>
   </property>
   <property name="text">
    <string>Superscript</string>
   </property>
   <property name="statusTip">
    <string>Superscript</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+U</string>
   </property>
  </action>
  <action name="actionStrikethrough">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/ico/Icons/Strikethrough.png</normaloff>:/ico/Icons/Strikethrough.png</iconset>
   </property>
   <property name="text">
    <string>Strikethrough</string>
   </property>
   <property name="statusTip">
    <string>Strikethrough</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionToolbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Toolbar</string>
   </property>
   <property name="statusTip">
    <string>Show/Hide Toolbar</string>
   </property>
  </action>
  <action name="actionStatus_Bar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Status Bar</string>
   </property>
   <property name="statusTip">
    <string>Show/Hide Status Bar</string>
   </property>
  </action>
  <action name="actionDisc">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/ico/Icons/Disc.png</normaloff>:/ico/Icons/Disc.png</iconset>
   </property>
   <property name="text">
    <string>Disc</string>
   </property>
   <property name="statusTip">
    <string>Create a Disc Bullet</string>
   </property>
  </action>
  <action name="actionCircle">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/ico/Icons/Circle.png</normaloff>:/ico/Icons/Circle.png</iconset>
   </property>
   <property name="text">
    <string>Circle</string>
   </property>
   <property name="statusTip">
    <string>Create a Circle Bullet</string>
   </property>
  </action>
  <action name="actionSquare">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/ico/Icons/Square.png</normaloff>:/ico/Icons/Square.png</iconset>
   </property>
   <property name="text">
    <string>Square</string>
   </property>
   <property name="statusTip">
    <string>Create a Square Bullet</string>
   </property>
  </action>
  <action name="actionArabic">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/ico/Icons/Arabic Numbers.png</normaloff>:/ico/Icons/Arabic Numbers.png</iconset>
   </property>
   <property name="text">
    <string>Arabic</string>
   </property>
   <property name="statusTip">
    <string>Create Arabic Number List</string>
   </property>
  </action>
  <action name="actionUppercase_Roman">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/ico/Icons/Uppercase Roman.png</normaloff>:/ico/Icons/Uppercase Roman.png</iconset>
   </property>
   <property name="text">
    <string>Uppercase Roman</string>
   </property>
   <property name="statusTip">
    <string>Create Uppercase Roman Number List</string>
   </property>
  </action>
  <action name="actionLowercase_Roman">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/ico/Icons/Lowercase Roman.png</normaloff>:/ico/Icons/Lowercase Roman.png</iconset>
   </property>
   <property name="text">
    <string>Lowercase Roman</string>
   </property>
   <property name="statusTip">
    <string>Create Lowercase Roman Number List</string>
   </property>
  </action>
  <action name="actionUppercase_Alphabetic">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/ico/Icons/Uppercase Alphabet.png</normaloff>:/ico/Icons/Uppercase Alphabet.png</iconset>
   </property>
   <property name="text">
    <string>Uppercase Alphabetic</string>
   </property>
   <property name="statusTip">
    <string>Create Uppercase Alphabetic List</string>
   </property>
  </action>
  <action name="actionLowercase_Alphabetic">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/ico/Icons/Lowercase Alphabet.png</normaloff>:/ico/Icons/Lowercase Alphabet.png</iconset>
   </property>
   <property name="text">
    <string>Lowercase Alphabetic</string>
   </property>
   <property name="statusTip">
    <string>Create Lowercase Alphabetic List</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
